src/components/Button/Button.tsx

```diff
@@ -2,33 +2,77 @@ import React from "react";
 import styled, { css } from "styled-components";
 
 export interface ButtonProps {
-  primary?: boolean;
   children: string;
+  variant: "filled" | "outlined";
+  theme: "light" | "dark" | "blue";
```

Comment:
I would rather use a theme object here, similar to this one
```js
colors: {
	'background': '#F4F7FB',
	'background-variant': '#E0EAFF',
	'surface': '#FFFFFF',
	'surface-variant': '#F9FBFD',
	'primary': '#0657F9',
	'primary-variant': '#004ADF',
	'secondary': '#E0EAFF',
	'titles': '#001746',
	'titles-variant': '#0657F9',
	'texts': '#7F8DA9',
	'icons': '#CED5E3',
	'icons-variant': '#475570'
}
```
This was we can abstract theming and allow anyone to use a custom theme

Reply:
Definitely, I just hardcoded those strings because I wasn't aware of https://github.com/yearn/yearn-web-themes/packages/1266161

Will refactor to use the theme now that I know the structure of the colors object

Reply:
Added the theme with https://github.com/Major-Eth/yearn-web-lib/pull/9/commits/efe8b82769ec5affe5430fc4ef9790f04c5db3d2

I think there are a few discrepancies between the theme and the design, for example, there's no `color` for the disabled state. I had to use `icons` -- maybe we can sync the theme with the design and the fix this.

Reply:
That's a good point
I am adding an issue on the repo to add missing ref names!